lib/hooks/bitbucket/templates/issue_created.html.haml

```diff
@@ -0,0 +1,17 @@
+:ruby
+  issue = payload.issue
+  actor = payload.actor
+
+%p
+  %span
+    = avatar_image_tag(actor.links.avatar.href)
+
+  %a{href: actor.links.html.href}
+    = actor.display_name
+  created issue to
+  %a{href: issue.links.html.href}
+    #{issue.title}##{issue.id}
```

Comment:
To keep consistency, `repository-full-name#issue-id issue-title` is better representation.

like [this](https://github.com/idobata/idobata-hooks/blob/725465bc0f721d4a02e29e71f43f6fb5d4a99125/lib/hooks/bitbucket/templates/pullrequest_created.html.haml#L12-L15):
![selection_353](https://cloud.githubusercontent.com/assets/43346/11261543/eca8ceae-8eb8-11e5-89de-0c59c706699a.png)